.github/workflows/benchmarks_gitlab.yml

```
@@ -22,20 +23,27 @@ jobs:
           toolchain: nightly
           override: true
 
-      - name: Set bench file
+      - name: Rust Cache
+        uses: Swatinem/rust-cache@v2.0.0
+
+      - name: Copy bench results
         id: step_one
         run: |
-          echo "bench_file=bench/$(date "+%d-%m-%Y")/output.txt" >> $GITHUB_ENV
+          cp bench/$(date "+%d-%m-%Y")/output.txt output.txt
```

nit: Could it be that the `$(date "+%d-%m-%Y")` leads to an inconsistent file between when:
 * (a) the bench file was published 
 * (b) this cronjob gets invoked?
 
 Ie, maybe we'll modify the cronjob's execution time and land one file behind?
 
 Something like the following might solve it, but didn't try it
 
 ```suggestion
           cp bench/$(ls -tA bench/ | head -n 1)/output.txt output.txt
```

I think another alternative would be for the gitlab script to also create a sym link `./bench/latest/output.txt` 😄

---

yeah, we could improve that but your suggestion doesn't work currently because they are other folders in bench as well

---

`bench/latest/output.txt` could work and our GHA could remove that file after producing the results successfully.

no issue with the date-format yet it was some issue that `gha action` switches to `gh-pages` and that's doesn't work if we already have switched to `gh-pages` branch.

that's it :P

---

@alvicsam https://github.com/paritytech/jsonrpsee/actions/runs/3190518964/jobs/5205730855 :tada: